rezasm-source/rezasm-core/src/instructions/implementation/function_instructions.rs

```diff
@@ -0,0 +1,47 @@
+use lazy_static::lazy_static;
+
+use crate::instructions::instruction::instruction;
+use crate::instructions::instruction::Instruction;
+use crate::instructions::instruction_registry::register_instruction;
+
+use crate::instructions::targets::input_output_target::InputOutputTarget;
+use crate::instructions::targets::input_target::Input;
+use crate::instructions::targets::input_target::InputTarget;
+use crate::parser::lexer::parse_line;
+use crate::parser::line::Line;
+use crate::util::error::{ParserError, SimulatorError};
+use crate::util::io::RezasmFileReader;
+use crate::util::raw_data::RawData;
+use crate::util::word_size::WordSize;
+
+lazy_static! {
+    pub static ref IMPORT: Instruction =
+        instruction!(import, |simulator: Simulator, input: InputTarget| {
+            let address = input.get(&simulator)?.int_value() as usize;
+            let file_path = simulator.get_memory().get_string(address)?;
+            let mut relative_location = simulator.get_program().main_file();
+            let relative_location_split = relative_location =
+                match relative_location.rsplit_once('/') {
```

Comment:
No windows support? Do not merge yet

Reply:
Windows does work -- `/` as a file separator works fine. On tauri right now you have to do something like `import "C:/Users/.../code.ez"` to import code on Windows, but that is good behavior and should be documented. We should be good to merge.